examples/interop/compose-in-uikit/iosApp/iosApp/Info.plist

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
Let's copy these lists from https://github.com/JetBrains/compose-multiplatform-template/blob/main/iosApp/iosApp/Info.plist

(it has CADisableMinimumFrameDurationOnPhone, for example)

Reply:
Done